server/src/test/java/io/crate/metadata/SchemasTest.java

```diff
@@ -192,46 +192,10 @@ public void testResolveTableInfoLooksUpRelationInSearchPath() throws IOException
         QualifiedName tableQn = QualifiedName.of("t");
         var sessionSettings = sqlExecutor.getSessionSettings();
         TableInfo tableInfo = sqlExecutor.schemas()
-            .resolveRelationInfo(tableQn, Operation.READ, sessionSettings.sessionUser(), sessionSettings.searchPath());
+            .findRelation(tableQn, Operation.READ, sessionSettings.sessionUser(), sessionSettings.searchPath());
 
         RelationName relation = tableInfo.ident();
         assertThat(relation.schema(), is("schema"));
         assertThat(relation.name(), is("t"));
     }
-
```

Comment:
Do we cover those cases elsewhere?

Reply:
`resolveRelationInfo`/`findRelation` has tests, yes and most code paths already used it.